web/packages/teleport/src/lib/tdp/client.ts

```diff
@@ -82,41 +82,41 @@ export default class Client extends EventEmitterWebAuthnSender {
   protected codec: Codec;
   protected socket: WebSocket | undefined;
   private socketAddr: string;
-  private width: number;
-  private height: number;
+  protected spec?: ClientScreenSpec;
   private sdManager: SharedDirectoryManager;
   private fastPathProcessor: FastPathProcessor | undefined;
 
   private logger = Logger.create('TDPClient');
 
-  constructor(socketAddr: string, width: number, height: number) {
+  // TODO(isaiah): width/height are optional -- include them when the client itself should determine the screen size such as in a desktop session.
```

Comment:
When do you plan to address this TODO? Looks like we already solved for this by taking an optional screen spec in connect?

Reply:
Yeah, this was a leftover comment